src/js/main.js

```diff
@@ -363,6 +363,29 @@ if (document.body.dataset.page === "integrations") {
   }
 }
 
+// Add underline class to menu links on Mobile
+
+function setActiveMenuItem() {
+  const currentUrl = window.location.href;
+  const menuItems = document.querySelectorAll("nav a");
+
+  menuItems.forEach((item) => {
+    const itemHref = item.getAttribute("href");
+    const itemUrl = new URL(itemHref, currentUrl);
+
+    const isIntegrationsPage = itemUrl.pathname === "/integrations";
+    const isInIntegrationsSection = currentUrl.includes("/integrations/");
+
+    const shouldBeUnderlined =
+      isIntegrationsPage &&
+      (isInIntegrationsSection || window.location.pathname === "/integrations");
+
+    item.classList.toggle("underline", shouldBeUnderlined);
```

Comment:
We can underling with border to make it a continuous line, which looks much nicer in my opinion :D

```suggestion
    item.classList.toggle("border-b-2 border-gray-600", shouldBeUnderlined);
```